Darwin/Services/StitchRemoteMongoDBService/StitchRemoteMongoDBService/ChangeStreamSession.swift

```diff
@@ -6,7 +6,7 @@ import StitchCoreRemoteMongoDBService
  * A MongoDB Change Stream session. This can be used to get or change the delegate that reacts to change events on
  * this stream. It can also be used to close open streams.
  */
-public class ChangeStreamSession<DelegateT: ChangeStreamDelegate> {
+public class ChangeStreamSession<T: Codable> {
```

Comment:
[nit] can you add a comment explaining what this type parameter does, or call it `DocumentT`?